packages/Checkbox/stories/examples/CheckboxGrouping.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
```

Comment:
What is this story demonstrating? (dont say "grouping")

Reply:
I think this is a relic of some a rough story I had committed.  I wanted to see the line-height and vertical spacing between `<Checkboxes>` at different sizes.  It's not quite doing that anymore, but 🤷‍♂️.  I regard it as a "dev" story that we can file away, delete, or glean something good from later for a curated "Variations" story (or something).